uni/lib/view/profile/widgets/print_info_card.dart

```diff
@@ -52,7 +52,18 @@ class PrintInfoCard extends GenericCard {
     );
   }
 
-  Widget addMoneyButton(BuildContext context) {
+  @override
+  String getTitle() => 'Impressões';
+
+  @override
+  onClick(BuildContext context) {}
+}
+
+class AddMoneyButton extends StatelessWidget {
```

Comment:
There is no need to extract small stateless widgets like this one, since there are no streamline in the parent's state and there is no benefit in testing this solo. In this case, you can simply inline it.